### ISSUE TITLE
Client missing parameters

### ISSUE THREAD

**Issue by nick-potts** [cloudflare/managed-component-to-cloudflare-worker#3]
The Client doesn't have access to some variables:
```
  screenHeight: string
  screenWidth: string
  viewportHeight: string
  viewportWidth: string
  timezoneOffset: string
  ```
  
  In other components, such as the GA4 component, they try retrieve this value. See:
  https://github.com/managed-components/google-analytics-4/blob/7e24d8ebf6880306f0d5076de6c5341ecdeb753e/src/requestBuilder.ts#L46
  
  I would make a pull request to fix, however this data doesn't exist in `event.payload`.

**Comment by bjesus** [comment]
will be fixed when https://github.com/cloudflare/managed-component-to-cloudflare-worker/pull/6 gets merged

**Comment by nick-potts** [comment]
What's the status on this? I'm seeing the right data getting passed in atm.

**Comment by bjesus** [comment]
yes, should be fixed now in version 1.4.0 !
